git-dit.1.md

```diff
@@ -95,6 +96,223 @@ the former one is especially useful for patch-sets.
+Various workflows are supported either directly or indirectly.
+This section discusses some possible workflows for managing issues using
+git-dit.
+
+## General setup considerations
+
+Project hosting sites often provide exactly one logical issue tracker for each
+project repository.
+As with mailing lists and stand-alone issue tracking web-services, users of
+git-dit are not forced to bind a set of issues to a specific source code
+repository.
+While Git-dit-issues do live in a git repository, it doesn't necessarily need to
+be the same repository used for the source code.
+For example, maintainers may choose to use separate repositories for code and
+issues.
+
+Consider, for example, a big project with several code repositories for
+different components.
+Now consider a user who wants to file a bug.
+In many cases, the user will not be able to correctly assign the bug to a
+specific component.
+With git-dit, maintainers may provide a dedicated issue repository for filing
+bug reports and feature requests.
+
+Contributors and sub-maintainers may add the central issue repository as an
+additional remote to their local clone.
+Optionally, issues assigned to a component may be transferred or copied to a
+component's code repository or an associated specialized issue repository
+(though there is no convenient support for such functionality, yet).
+
+It is also possible to have multiple issue repositories for a single code
+repository.
+Consider for example one of these closed source projects with many change
+requests from difficult customers.
+With git-dit, you can set up a dedicated issue repository for each customer in
+order to accept and share change requests.
+Developers can add each of those issue repositories as remotes, viewing and
+interacting with all of the issues from different customers.
+
+## Retrieving issues from a remote repository
+
+Naturally, contributors as well as maintainers will which to retrieve issues
+a remote repository, be it a dedicated issue repository or a repository
+containing both issues and source code.
+Git-dit offers multiple ways for retrieving issues.
+
+Issues may be fetched manually from a remote repository using the `fetch`
+subcommand.
+
+    git dit fetch issue-repo
+
+will fetch all issues from a remote "issue-repo". The subcommand also supports
+fetching only updates of issues explicitly specified or all issues which are
+known to the current repository.
+
+Alternatively, a developer may choose to subscribe to the issues present in a
+remote repository.
+Currently, this has to be done manually through a ref-spec.
+The following refspec, for example, will cause the issues to be fetched from the
+remote "origin" on each `git fetch`.
+
+    refs/dit/*:refs/remotes/origin/dit/*
+
+New issues and issue updates are pushed to a remote using the `push` subcommand.
+
+## Reporting an issue
+
+Issues can be created in the local repository.
+
+    git dit new
```

Comment:
Just as an idea: Maybe we should have an alias "new-issue"... semantically it would make sense... "git dit new" does not express _what_ will be created. We could use "new" as shortcut for "new-issue"... I do not have strong feelings on this, though.